posthog/cdp/templates/avo/template_avo.py

```diff
@@ -124,3 +125,51 @@
         },
     ],
 )
+
+
+class TemplateAvoMigrator(HogFunctionTemplateMigrator):
+    plugin_url = "https://github.com/PostHog/posthog-avo-plugin"
+
+    @classmethod
+    def migrate(cls, obj):
+        hf = deepcopy(dataclasses.asdict(template))
+
+        apiKey = obj.config.get("avoApiKey", "")
+        environment = obj.config.get("environment", "dev")
+        appName = obj.config.get("appName", "PostHog")
+        excludeEvents = obj.config.get("excludeEvents", "")
+        includeEvents = obj.config.get("includeEvents", "")
+        excludeProperties = obj.config.get("excludeProperties", "")
+        includeProperties = obj.config.get("includeProperties", "")
+
+        hf["filters"] = {}
+        hf["filters"]["events"] = []
+
+        events_to_include = [event.strip() for event in includeEvents.split(",") if event.strip()]
+        events_to_exclude = [event.strip() for event in excludeEvents.split(",") if event.strip()]
+
+        if events_to_include:
+            hf["filters"]["events"] = [
+                {"id": event, "name": event, "type": "events", "order": 0} for event in events_to_include
+            ]
+        elif events_to_exclude:
+            event_string = ", ".join(f"'{event}'" for event in events_to_exclude)
```

Comment:
You can use this function to be really sure all is good:

```suggestion
            event_string = ", ".join(escape_hogql_string(event) for event in events_to_exclude)
```